builtin/staker/housekeep.go

```diff
@@ -247,3 +265,58 @@ func (s *Staker) activateNextValidation(currentBlk uint32, maxLeaderGroupSize ui
 
 	return validator, nil
 }
+
+// This check ensures that locked + queued + withdrawable VET equals the total VET in the staker account address.
+func (s *Staker) PerformSanityCheck() error {
```

Comment:
nit: Can this name be changed to something like `BalanceCheck` or `ContractBalanceCheck` or something a bit more descriptive ?